grpc/flatbuffers-js-grpc/src/client.ts

```diff
@@ -0,0 +1,20 @@
+import grpc from 'grpc';
+import { HelloRequest } from './greeter_generated';
+import services from './greeter_grpc';
+
+const flatbuffers = require('flatbuffers').flatbuffers;
+
+function main() {
+    console.log(services);
+    let _server = new services.GreeterClient(`localhost:${3000}`, grpc.credentials.createInsecure());
+    let builder = new flatbuffers.Builder();
```

Comment:
There is an issue with the `var _server = new services.GreeterClient()` if I use the build typescript command it will always convert it to `var _server = new hello_grpc_1.default.GreeterClient()` which becomes and issue since this is how the export function works `exports.GreeterClient = grpc.makeGenericClientConstructor(GreeterService);` how can I fix this in a proper way?

Reply:
I don't really understand what the issue is. Can you provide more details what's wrong?

Reply:
I fixed it :D it was an import issue